octoprint_octolapse/data/settings_default.json

```diff
@@ -2145,6 +2145,14 @@
       "name": "FLV",
       "value": "flv"
     },
+    {
+      "name": "GIF",
+      "value": "gif"
+    },
+    {
+      "name": "H.264",
```

Comment:
Maybe we should indicate in the name here that the video will be rendered as an MP4 to make it clear?  Maybe Name = MP4 - H.264?  We could also update the other MP4 option too so that it shows what codec is being used.

Reply:
Fixed.